packages/twenty-front/src/modules/views/components/UpdateViewButtonGroup.tsx

```diff
@@ -33,9 +35,13 @@ export type UpdateViewButtonGroupProps = {
 export const UpdateViewButtonGroup = ({
   hotkeyScope,
 }: UpdateViewButtonGroupProps) => {
-  const { canPersistViewSelector, currentViewIdState } = useViewStates();
+  const { canPersistViewSelector } = useViewStates();
   const { saveCurrentViewFilterAndSorts } = useSaveCurrentViewFiltersAndSorts();
 
+  const currentViewIdState = useExtractedComponentState(
```

Comment:
I would keep calling it: useRecoilComponentState

Exact same API as a regular recoilState but we name everything ComponentState

Reply:
It's not the same API, useRecoilComponentState subscribes and useExtractedComponentState only returns the state for useRecoilCallback, maybe we can rename later to precise this ? 

`currentViewIdCallbackState` ?